configs/test/bigquery/datasets.yaml

```diff
@@ -225,3 +225,29 @@ resources:
             type: FLOAT
           - name: engine
             type: STRING
+  - name: cross-pollination-statistics-table
+    type: gcp-types/bigquery-v2:tables
+    properties:
+      datasetId: $(ref.main-dataset.datasetReference.datasetId)
+      tableReference:
+        tableId: cross-pollination-statistics
+      timePartitioning:
+        type: DAY
+      schema:
+        fields:
+        - name: project_qualified_name
+          type: STRING
+        - name: tagged
+          type: BOOLEAN
+        - name: original_corpus_size
```

Comment:
sorry for the nit picking, I'd propose the following naming convention:
- `initial_corpus_size` for the "original" values
- `corpus_size` for the post pollination values

and the same for the coverage metrics. That's more similar to what we use in the existing stats table.